README.md

```diff
@@ -1 +1,50 @@
-# restaurant-page
\ No newline at end of file
+# Restaurant Page
+
+> This is a restaurant one page with interactive menu.
+
+![screenshot](src/img/screenshot.png)
+
+This project has these features:
+  - One page menu.
+  - Dinamically changes menus without sliding.
+  - Classic design.
+
+## Built With
+
+- HTML, CSS
+- Javascript
+- Webpack
+
+## Settings
+
+- Run `npm install` to have all the needed packages to run the app.
+
+## Live Demo
+
+[Live Demo Link](https://majovanilla.github.io/restaurant-page
```

Comment:
Let make our Repo looks professional by adding a live link.
Kindly watch the video tutorial on how to create a live link 👉  [Link](https://www.loom.com/share/76088842a2d14322bbcf65b884f4d42d);

Reply:
The live version link is set and ready. I use GitHub pages and the link will work ONCE I merge with master.

Reply:
I used `raw.githack` with your repo and everything works fine, but make sure to use `GitHub pages` later on.